Src/PerformanceCollector/Shared/QuickPulseTelemetryProcessor.cs

```diff
@@ -178,55 +184,96 @@ private static ITelemetryDocument ConvertRequestToTelemetryDocument(RequestTelem
+                           Id = Guid.NewGuid(),
                            Version = TelemetryDocumentContractVersion,
                            Timestamp = dependencyTelemetry.Timestamp,
-                           Id = dependencyTelemetry.Id,
-                           Name = dependencyTelemetry.Name,
+                           Name = TruncateValue(dependencyTelemetry.Name),
                            StartTime = dependencyTelemetry.StartTime,
                            Success = dependencyTelemetry.Success,
                            Duration = dependencyTelemetry.Duration,
-                           Sequence = dependencyTelemetry.Sequence,
                            ResultCode = dependencyTelemetry.ResultCode,
-                           CommandName = dependencyTelemetry.CommandName,
+                           CommandName = TruncateValue(dependencyTelemetry.CommandName),
                            DependencyTypeName = dependencyTelemetry.DependencyTypeName,
-                           DependencyKind = dependencyTelemetry.DependencyKind
+                           DependencyKind = dependencyTelemetry.DependencyKind,
+                           Properties = GetProperties(dependencyTelemetry, SpecialDependencyPropertyName)
                        };
         }
 
         private static ITelemetryDocument ConvertExceptionToTelemetryDocument(ExceptionTelemetry exceptionTelemetry)
         {
-            // //!!! cut length for Exception.ToString()
             return new ExceptionTelemetryDocument()
                        {
+                           Id = Guid.NewGuid(),
                            Version = TelemetryDocumentContractVersion,
-                           Message = exceptionTelemetry.Message,
+                           Message = TruncateValue(exceptionTelemetry.Message),
                            SeverityLevel =
                                exceptionTelemetry.SeverityLevel != null
                                    ? exceptionTelemetry.SeverityLevel.Value.ToString()
                                    : null,
                            HandledAt = exceptionTelemetry.HandledAt.ToString(),
                            Exception =
-                               exceptionTelemetry.Exception != null ? exceptionTelemetry.Exception.ToString() : null
+                               exceptionTelemetry.Exception != null
+                                   ? TruncateValue(exceptionTelemetry.Exception.ToString())
+                                   : null,
+                           ExceptionType =
+                               exceptionTelemetry.Exception != null
+                                   ? TruncateValue(exceptionTelemetry.Exception.GetType().FullName)
+                                   : null,
+                           ExceptionMessage =
+                               exceptionTelemetry.Exception != null
+                                   ? TruncateValue(exceptionTelemetry.Exception.Message)
```

Comment:
>Message [](start = 80, length = 7)

exceptionTelemetry has Message property that by default is exception.Message but customer may override it to smth more meaningful